README.MD

```diff
@@ -1,8 +1,8 @@
 ## MENTORS-FRONTEND
 
-### Environemnt Setup
+### Environment Setup
 #### Install dependencies 
- * npm
+ * npm install
```

Comment:
I wanted to make sure people will install npm. 
So we need to add a line indicate to run npm install before npm start.

Reply:
ill add prerequisites, good idea

Reply:
Great.